test/games_collection_test.rb

```diff
@@ -104,6 +104,6 @@ def test_it_can_return_hash_of_average_goals_by_season
                       "20152016"=>3.88,
                       "20132014"=>4.33
                     }
-    assert_equal expected_hash, @games_collection.average_goals_by_season
+    assert_equal expected_hash, @games_collection.average_goals_per_season
```

Comment:
The method name that Iteration 2 calls for is `average_goals_by_season` - if you want to discard/revert the changes on this file, my "quick fix" pull request updates it to the way the spec harness expects.

Reply:
done